### ISSUE TITLE
Question only - How to install

### ISSUE THREAD

**Issue by harryinet** [blakadder/esphome-configs#1]
Hi, may i copy the yaml and paste into my esphome athom device? Do i need the api encrytion?

**Comment by blakadder** [comment]
Yes but take care the names are the same. I don't know if you need the encryption, I don't use it

**Comment by harryinet** [comment]
Yes, thanks, at last i only changed the URL, saved and installed, that works fine. I just started to use the Esphome. Thank you for your improvement.
